app/Service/GithubServiceProvider.php

```diff
@@ -75,6 +79,9 @@ public function getSponsorsList(): array
 
         if ($response['code'] == 200) {
             $data = json_decode($response['body'], true);
+            if(true) {
```

Comment:
I don't understand this if here!

Reply:
Upsi.. It's a for testing that I forgot to remove. Will take care when back from vacation.

Reply:
It was a faulty commit from testing. I have reverted that part.